apps/test/util/runLevelTest.js

```diff
@@ -20,7 +19,7 @@ module.exports = function(testCollection, testData, dataItem, done) {
   if (!level.scale) {
     level.scale = {};
   }
-  level.scale.stepSpeed = 0;
+  level.scale.stepSpeed = 1;
```

Comment:
Step speed of 0 resulted in us calling setInterval(fn, 0), which sinon did not handle well (looped infinitely).